geoportal/vars.yaml

```
@@ -868,6 +868,9 @@ vars:
         - https://127.0.0.1:3000
         - https://127.0.0.1:3001
         - '*'
+        # GeoGirafe
+        - 'https://app.localhost:8080'
+        - 'https://demo.geomapfish.dev'
```

Don't use unneeded quote
```suggestion
        - https://app.localhost:8080
        - https://demo.geomapfish.dev
```